spec/requests/requests_requests_spec.rb

```diff
@@ -74,6 +74,57 @@
           expect(response.body).not_to include('Default storage location inventory')
         end
       end
+
+      context 'When packs are enabled' do
+        before { Flipper.enable(:enable_packs) }
+        let(:item) { create(:item, name: "Item", organization: organization) }
+        let(:request) { create(:request, organization: organization) }
+
+        it 'shows a units column if any item has custom units' do
+          create(:item_unit, item: item, name: "Pack")
+          create(:item_request, request: request, request_unit: "Pack", item: item)
+
+          get request_path(request)
+
+          expect(response.body).to include('Units (if applicable)')
+        end
+
+        it 'does not show a units column if no items have custom units' do
```

Comment:
We can combine these tests - there's no need to have only a single expectation per test. In this case there are two tests - one where no items have units and one where at least one item has a unit.

Reply:
Ah got it. The column won't show up unless at least one item has a unit so I see how they would go hand in hand. The identical test setup should've been my hint. Thanks!